src/express/types.ts

```diff
@@ -24,10 +24,15 @@ export declare type PayloadRequest<U = any> = Request & {
    * - Configuration from payload-config.ts
    * - MongoDB model for this collection
    * - GraphQL type metadata
-   * */
+   */
   collection?: Collection;
   /** What triggered this request */
   payloadAPI?: 'REST' | 'local' | 'GraphQL';
+  /** payloadContext allows you to pass your own data to the request object as context
+   * This is useful for, for example, passing data from a beforeChange hook to an afterChange hook.
+   * payoadContext can also be fully typed using declare module (more info in the docs)
+   */
+  payloadContext: PayloadRequestContext;
```

Comment:
What is the thinking behind calling this `payloadContext` over just `context`?

Reply:
> `req.context`

☝🏼 💯 👍🏼

Reply:
> What is the thinking behind calling this payloadContext over just context?

The idea was that that way, it makes clear that this is payload functionality and not express functionality. With just `context`, it becomes less clear that this is something added by payload.

On the other hand, yeah, naming it just `context` is shorter and cleaner. I wouldn't be opposed to changing it.

Reply:
The real reason I like `context` better is that `payloadContext` could be confused to mean that it is for internal to Payload's own use, not meant for your project's code.